website/docs/guides/1.4.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Let's remove this out of this PR since it's in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/747

Reply:
Removed